spyder/app/tests/test_mainwindow.py

```diff
@@ -1465,4 +1476,4 @@ def test_render_issue():
 
 
 if __name__ == "__main__":
-    pytest.main()
+    pytest.main(['-x', os.path.basename(__file__), '-v', '-rw'])
```

Comment:
Is there is a reason this was changed, particularly adding the ``-x`` option? In general, when running locally I usually use ``-v`` (or ``-vv``) and always use ``-rw``, but almost never want ``-x`` and would have to routinely modify the ``runtests.py`` file to change this if I wanted to run the whole test suite with this.

Reply:
Running the whole suite when trying to make a single test work is not very efficient. I forgot to exclude this change when committing, I'll simply revert in another commit.